src/sid/contacts.py

```diff
@@ -55,6 +55,8 @@ def calculate_contacts(
 
         if model["is_recurrent"]:
             model_specific_contacts = model_specific_contacts.astype(bool)
+        else:
```

Comment:
I am not sure if I like it. Of course, we can prevent some unnecessary errors, but at the same time it conflicts with a user's expectation if she created an integer. Even worse, memory consumption is increased. Maybe we could make it optional if the dtype is `object` or fix sid-germany altogether

Reply:
I will address this in a new PR.